tests/data_association/test_data_association.py

```diff
@@ -80,6 +82,19 @@ def generate_noisy_2d_measurements(
     return keypoints_list, img_idxs, cameras
 
 
+def get_camera_details(
```

Comment:
is this needed? generate_noisy_2d_measurements above can be modified to output the wRi, wti and intrinsics. 

There are one or two places where cameras are created without calling the above method. In those cases we directly create wRi, wti and intrinsics instead.